web/wp-content/themes/dctx-v2/sidebar.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-cmvcwh.php, but found phpcs-scan-cMvcWH.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).